content/docs/user-guide/networking/settings.md

```diff
@@ -0,0 +1,179 @@
+---
+title: Networking And Multiplayer Settings
+description: A reference for console variables and other settings that can be used on clients and servers to configure networking and multiplayer.
+linktitle: Auto-packets
+---
+
+## Overview
+This page documents console variables (CVars) and settings for AzNetworking and Multiplayer behaviour.
+
+## Client settings
+| Setting                | Description                                                                                                                                                                            | Default   | Notes |
+|------------------------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|-----------|-------|
+| cl_clientport          | The port to bind to for game traffic when connecting to a remote host, a value of 0 will select any available port                                                                     | 0         |       |
```

Comment:
All descriptions should use sentence punctuation, i.e. end with a period (".").

Reply:
Will fix.